.packit.yaml

```
@@ -0,0 +1,29 @@
+specfile_path: ansible-lint.spec
+actions:
+  create-archive:
+    - "python3 setup.py sdist --dist-dir ."
+    - "sh -c 'echo ansible-lint-$(python3 setup.py --version).tar.gz'"
+  get-current-version:
+    - "python3 setup.py --version"
+  post-upstream-clone:
+    - |
+      bash -c "sed -i -e 's/Version: VERSION_PLACEHOLDER/Version: $(python3 setup.py --version)/' ansible-lint.spec"
+jobs:
```

If we modify files in the repo, this will confuse setuptools-scm. So it's best to pretend like nothing happened:
```suggestion
    - git update-index --assume-unchanged ansible-lint.spec
jobs:
```